.github/workflows/docker-base.yml

```diff
@@ -46,15 +41,19 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
+        os:
+          - tag: ubuntu22.04
```

Comment:
The tag could be built by os and os_version, similar to the file-dir if its technically possible by github
```
os: ubuntu|alpine
os_version: 22.04|3.19
tag: ${{ matrix.os }}${{ matrix.os_version }}
file-dir: src/docker/base/${{ matrix.os }}
```

Reply:
That is unfortunately not possible in github workflows.